packages/repo.rb

```diff
@@ -0,0 +1,33 @@
+
+require 'package'
+ 
+class Repo < Package
+  description 'Android Repo'
+  homepage 'https://gerrit.googlesource.com/git-repo/'
+  version '1.0'
+  license 'Apache License 2'      
+  compatibility 'x86_64' 
+  source_url 'https://storage.googleapis.com/git-repo-downloads/repo'
+  source_sha256 '974992a8bc1c787979f3eb7702a803b051deddc3cd75726c4de52e09e93b798d '
+
+  depends_on 'gpgme'
```

Comment:
How is this used?

Reply:
gpgme is for gpg key

Reply:
Right, I guess my question is how is gpg used for the install?

Reply:
for verity repo
export REPO=$(mktemp /tmp/repo.XXXXXXXXX)
curl -o ${REPO} https://storage.googleapis.com/git-repo-downloads/repo
gpg --recv-key 8BB9AD793E8E6153AF0F9A4416530D5E920F5C65
curl -s https://storage.googleapis.com/git-repo-downloads/repo.asc | gpg --verify - ${REPO} && install -m 755 ${REPO} ~/bin/repo
from
https://source.android.com/docs/setup/develop?hl=it#installing-repo

Reply:
Doesn't `source_sha256` do that too?